Loop/Managers/RemoteDataServicesManager.swift

```diff
@@ -586,15 +586,13 @@ extension RemoteDataServicesManager {
 
 extension RemoteDataServicesManager {
 
```

Comment:
As mentioned above, we will validate a push notification by the service that generated it.